articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,186 @@
+
+However, your Angular application needs to pass an access token when it calls a target API to access private resources. You can [request an access token](https://auth0.com/docs/tokens/guides/get-access-tokens) in a format that the API can verify by passing the `audience` and `scope` props to `AuthModule.forRoot`.
+
+In addition, the configuration should specify which requests should have an `Authorization` header added to them with the access token.
+
+The following example shows how to set up your Angular application module, with all the necessary imported types and the configuration to call the management API:
+
+```javascript
+// Import the injector module and the Angular types you'll need
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { AuthHttpInterceptor } from '@auth0/auth0-angular';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    BrowserModule,
+
+    // Register HttpClient for making API calls
+    HttpClientModule,
+    
+    AppRoutingModule,
+    AuthModule.forRoot({
+      domain: "${account.namespace}",
+      clientId: "${account.clientId}",
+
+      // Specify the audience for the management API, allowing us to make API calls
+      audience: "https://${account.namespace}/api/v2/",
+
+      // Specify the necessary scopes for getting user information
+      scope: "read:current_user",
+
+      // Specify configuration for the interceptor              
+      httpInterceptor: {
+        allowedList: [
+          {
+            // Match any request that starts 'https://${account.namespace}/api/v2/' (note the asterisk)
+            uri: 'https://${account.namespace}/api/v2/*'
+            tokenOptions: {
+              // The attached token should target this audience
+              audience: 'https://${account.namespace}/api/v2/',
+
+              // The attached token should have these scopes
+              scope: 'read:current_user'
+            }
+          }
+        ]
+      }
+    }),
+  ],
+  providers: [
+    // Register the `AuthHttpInterceptor` to automatically attach the 'Authorization' header
+    { provide: HTTP_INTERCEPTORS, useClass: AuthHttpInterceptor, multi: true },
+  ],
+  bootstrap: [AppComponent],
+})
+```
+
+Auth0 uses the value of the `audience` prop to determine which resource server (API) the user is authorizing your Angular application to access.
+
+Notice the `httpInterceptor` configuration, which specifies:
+
+* Any request where the URL starts with `https://${account.namespace}/api/v2/` should have an access token attached
```

Comment:
longer term comment.
hmm, `account.namespace` looks odd... I am sure you are using this to be consistent with what we have done elsewhere: given that in other SDKs we use different terms (domain? tenant?) it might be worth to do a wide sweep to track what we use across the board, and begin a consolidation effort